cmd/erasure-server-pool-decom.go

```diff
@@ -95,7 +109,7 @@ func (pd *PoolDecommissionInfo) isBucketDecommissioned(bucket string) bool {
 	return false
 }
 
-func (pd *PoolDecommissionInfo) bucketPush(bucket string) {
+func (pd *PoolDecommissionInfo) bucketPush(bucket string, prefix string) {
```

Comment:
It is only going to list specific paths provided by prefix.

Reply:
Because we are only interested in `decommission` of content in `config/` and `buckets/`  from `.minio.sys` -  we are not interested in any other objects in that folder @vadmeste